tests/core/training/test_story_conflict.py

```diff
@@ -131,6 +131,20 @@ async def test_find_conflicts_multiple_stories():
     assert "and 2 other trackers" in str(conflicts[0])
 
 
+async def test_find_unpredictable_actions():
+    trackers, domain = await _setup_trackers_for_testing(
+        "data/test_domains/default.yml",
+        "data/test_yaml_stories/stories_unpredictable.yml",
```

Comment:
@alwx I see that you removed this [file](https://github.com/RasaHQ/rasa/pull/8913/commits/2203d2345a7b8e9705358563fbaad90ceed97293) in the last commit of yours but the test still searches for it leading it to fail. Can you please check?

Reply:
Updated it already.

Reply:
Thanks. Just a small note: Please don't use `intent_ted` to refer anything. We have renamed the policy to `UnexpecTEDIntentPolicy`. Reverted some of the changes from your commit in this [commit ](https://github.com/RasaHQ/rasa/pull/8913/commits/575d87f258a4fca6293f35d6ba173ac58fac2b2e) of mine